### PR TITLE
Update contributing for single .flake8 file

### PR DESCRIPTION
**Pull request by peterjc**
See PR #2074 and issues #493 / #2014

This pull request addresses issue #...

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -53,24 +53,19 @@ Unless you are using Python 2.7, please also install the bugbear plugin:
...
-    $ flake8 BioSQL/
-    $ flake8 Tests/
-    $ flake8 Scripts/
-    $ flake8 Doc/examples/
-
-Each of these folders has its own flake8 settings file, but in the long term
-we would like all the code to follow the same strict settings.
+    $ flake8
 
 You can run the reStructuredText checks with the ``restructuredtext-lint``
```
Comment (peterjc) [inline CONTRIBUTING.rst]: Maybe move this after the flake8 hook bit (pending a more sophisticated hook setup)?

Comment (MarkusPiotrowski) [reply CONTRIBUTING.rst]: I would suggest to move everything from line 45-52 (installing flake8) directly below line 36, lines 64-68 (flake8 pre-commit hook) directly below that (with a minor rephrase), and then mention the flake 8 command line access. Finally the other stuff.
Like this: 
1. "We follow PEP-8 and PEP-257"
2. "You install flake8"
3. "You set up a pre-commit hook"
4. "You can also run flake8 independently"
5. "For docstrings, we are also using...."
....

Comment (peterjc) [reply CONTRIBUTING.rst]: Sounds good - feel free to update this branch if I’ve not, but will try tomorrow

**Comment by peterjc** [comment]
That's nicer, thank you.

**Pull request merged**